lib/kubernetes-deploy/kubernetes_resource/service_account.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+module KubernetesDeploy
+  class ServiceAccount < KubernetesResource
+    TIMEOUT = 30.seconds
+
+    def sync
+      _, _err, st = kubectl.run("get", type, @name, "--output=json")
+      @found = st.success?
+    end
+
+    def deploy_succeeded?
+      exists?
+    end
+
+    def deploy_failed?
+      false
+    end
```

Comment:
We should probably use 
```ruby
    def timeout_message
      UNUSUAL_FAILURE_MESSAGE
    end
```
here (like for configmap) since this shouldn't ever time out